ci/test_wheel_distributed_ucxx.sh

```diff
@@ -25,8 +25,6 @@ rapids-logger "Install Distributed in developer mode"
 git clone https://github.com/dask/distributed /tmp/distributed
 python -m pip install -e /tmp/distributed
 
-print_ucx_config
```

Comment:
@vyasr  looks ok for wheels, but can you remind why `ucx_info` isn't available in the wheel package ? This is very often required for any debugging, so I think we should always include it.

Reply:
There isn't an easy way to bundle arbitrary executables into the wheel. It can be done, but it'll require more of the type of complexity that we're already doing for bundling ucx libraries, along with some extra bits to wrap that in an installable Python entry point. I don't think we want to gate this PR on figuring that out.

Reply:
We definitely don't want that in this PR, but this may be something we should consider looking into sometime in the near future. Any serious-ish use of UCX will eventually need that information for some debugging.